src/lib/ng-v5/entry-point/resources/stylesheet-processor.ts

```diff
@@ -10,9 +10,16 @@ import * as nodeSassTildeImporter from 'node-sass-tilde-importer';
 import * as postcss from 'postcss';
 import * as postcssUrl from 'postcss-url';
 import postcssClean from './postcss-clean';
-import * as sass from 'node-sass';
 import * as stylus from 'stylus';
 
+let sassComplier: any | undefined;
+try {
+  // tslint:disable-next-line:no-implicit-dependencies
+  sassComplier = require('sass'); // check for dart-sass first
+} catch {
+  sassComplier = require('node-sass'); // check for dart-sass first
```

Comment:
The comment is a copy-paste mistake.

Also, now that `node-sass` is being marked as optional it may happen that it is not available. Users without `sasss` (Dart sass) will then get a nasty warning. We should allow this `require` to fail as well, and deal with an `undefined` `sassCompiler` at the point where we actually need it; e.g. by providing the user with useful information on why it fails and what could be done to fix it. That makes the `node-sass` dep really optional and also helps developers to find that they either need to install `node-sass`, or use Dart's `sass` when `node-sass` is not an option for them.